content/v2/pedagogy/syllabus.md

```diff
@@ -0,0 +1,146 @@
+
+Gofer Grader works with the same API that `OKpy` uses, meaning that using one package
+or another should require no change in the code itself. The only difference is that
+Gofer Grader will not send any course material to the OKpy servers.
+
+> **Note:** `GoferGrader` and `OKpy` should not both be installed at the same time. If one
+  is installed, the other should be uninstalled.
+
+### Customizing your own copies of the homework
+
+To customize the Data 8 homework for your course, take the following steps:
+
+1. Fork and clone the Data 8 materials for the semester of your choice (see above section on labs).
+2. Ensure that you have a JupyterHub running the Data 8 environment (if not, [follow these instructions](customize_hub_environment.html))
+3. Create your interact links so they point to *your* version of the homeworks. For example,
+
+       http://<< YOUR-HUB-URL >>/hub/user-redirect/git-sync?repo=https://github.com/<< YOUR-USERNAME >>/materials-sp18/materials/sp18/hw/hw07/hw07.ipynb
+
+## Labs
+
+Alongside the textbook are several computational labs that let students interact with the
+ideas covered in class. They can all be run interactively in the Data 8 environment (the same
+environment that we've used in your JupyterHub).
+
+The latest links to the labs can be found in the course page for the current
+semester. For example, the Spring 2018 course is found here:
+
+[http://data8.org/sp18/]
+
+These materials are freely available on the semester course repository.
+There is a new repository created for each semester of the course, as the materials
+tend to evolve over time. For example, here is the repository for the Spring 2018 course:
+
+[https://github.com/data-8/materials-sp18]
+
+you can find the labs for this repository here:
+
+[https://github.com/data-8/materials-sp18/tree/master/materials/sp18/lab]
+
+
+### Customizing your own copies of the labs
+
+All labs should already be runnable on the environment that the Data 8 Docker image serves.
+The Data 8 materials are shared by adding
+"interact links" to the following syllabus page:
+
+[http://data8.org/sp18/]
+
+We recommend forking and customizing your own version of the labs, so that you can ensure the
+content is the right match for your course. To do this, perform the following steps:
+
+1. Fork the repository for the course you want to run. (e.g., `https://github.com/data-8/materials-sp18/`)
+2. Clone your repository to your machine
+
+       git clone https://github.com/<YOUR-USERNAME>/materials-sp18/
+
+3. Make modifications to the labs as you see fit, push them to GitHub
+4. Ensure that you have a JupyterHub running the Data 8 environment (if not, [follow these instructions](customize_hub_environment))
+5. Create your interact links so they point to *your* version of the labs. For example,
+
+       http://<< YOUR-HUB-URL >>/hub/user-redirect/git-sync?repo=https://github.com/<< YOUR-ORG >>/materials-sp18&subPath=materials/sp18/lab/lab02/lab02.ipynb
+
```

Comment:
Do you think we need this section? It's almost identical to the homeworks section.